api/src/lib/aws.ts

```diff
@@ -108,8 +108,7 @@ export async function s3PutSignedUrl(
   const baseParams: PutObjectCommandInput = {
```

Comment:
Sorry am very late to this since I was out last week but, do we need to update this path now that we don't have `organizationId` directly on `upload` anymore?